### PR TITLE
2023 Development Roadmap

### PR DESCRIPTION
**Pull request by robfalck**
### Summary

Adds a list of goals and general development path for 2023.

### Related Issues

- None

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: roadmaps/2023roadmap.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+We were pleased to host the first in-person OpenMDAO workshop since 2019.
+This provided the community with a look at how various organizations are using OpenMDAO, and provided valuable feedback for us as developers.
+We get to see ways in which the tool has enabled some really impressive work by the community.
+We also get to learn about challenges the community faces and develop ways to overcome these challenges.
+
+It's fair to say that OpenMDAO has reached a level of maturity.
+The fundamental capability is sound, and through some iteration we've arrived at a stable API.
+With a strong, technically sound foundation much of our focus in the next year will be on continuing to increase the productivity of users by removing friction where we can.
+
+- Our goal of developing an interactive training course has produced a great series of [Practical MDO](https://www.youtube.com/@OpenMDAO/featured) videos and corresponding notebooks.
```
Comment (swryan) [inline roadmaps/2023roadmap.md:25]: suggest linking to the PracticalMDO [playlist](https://www.youtube.com/playlist?list=PLPusXFXT29sXfMEFH3Czj78SrjNeY4M4-)  (featured is currently showing the workshop videos)

File: roadmaps/2023roadmap.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+When providing analytic derivatives, this usually involves some referencing of textbooks or online differentiation tools.
+
+### Goal
+
+Rather than having the user go through this friction often, we will provide a set of common mathematical routines as
+functions along with functions that provide their derivatives.
+This should make it significantly easier for users to put together their own derivatives in `compute_partials`.
+
+While we considered automatic differentiation tools, that involves bringing in some significant dependencies and in
+some cases the tools aren't capable.  For instance, `jax.grad` will only provide derivatives for scalar outputs.
```
Comment (swryan) [inline roadmaps/2023roadmap.md:117]: clarity: capable of what?  or did you mean `compatible`?

File: roadmaps/2023roadmap.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- We continue to move away from reliance on feedback through the terminal and instead provide richer feedback through reports. We've received positive feedback on the standard set of reports we generate and will continue to improve upon these and develop new ones when warranted.
+- The push to bring 2nd-derivative information into the Modular Approach to Unified Derivatives (MAUD) theory upon which OpenMDAO is based remains an open goal but there is still interest in that area.
+ 
+## 2023 Focus Areas 
+
+A broad goal for OpenMDAO is to make things simple for ~80% of our users who are doing typical analysis, while making things possible for the ~20% of our users who are advanced.
+
+In the past we've always said that there are ways for users to incorporate OpenMDAO into their workflows.
+In practice this means that users developing different tools somtimes come together and find that they need to rework their implementations so that their codes are compatible.
```
Comment (swryan) [inline roadmaps/2023roadmap.md:35]: typo: somtimes

File: roadmaps/2023roadmap.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+- Work on the [Mphys](https://github.com/OpenMDAO/mphys) package has pushed our capability in terms of large models involving parallel computing. This work resulted in a [significant API change](https://github.com/OpenMDAO/POEMs/blob/master/POEM_075.md) to make the use of distributed components more clear, with some more improvements on the way.
+- We continue to move away from reliance on feedback through the terminal and instead provide richer feedback through reports. We've received positive feedback on the standard set of reports we generate and will continue to improve upon these and develop new ones when warranted.
+- The push to bring 2nd-derivative information into the Modular Approach to Unified Derivatives (MAUD) theory upon which OpenMDAO is based remains an open goal but there is still interest in that area.
+ 
+## 2023 Focus Areas 
+
+A broad goal for OpenMDAO is to make things simple for ~80% of our users who are doing typical analysis, while making things possible for the ~20% of our users who are advanced.
```
Comment (Kenneth-T-Moore) [inline roadmaps/2023roadmap.md:32]: Do 80% and 20% come from somewhere?

**Review by Kenneth-T-Moore — APPROVED** [review]

File: roadmaps/2023roadmap.md (hunk 4 above)
Comment (robfalck) [reply roadmaps/2023roadmap.md:32]: Only from a general [80/20 rule](https://en.wikipedia.org/wiki/Pareto_principle)

**Review by swryan — APPROVED** [review]

**Pull request merged**